packages/fiori/src/i18n/messagebundle.properties

```diff
@@ -98,7 +98,13 @@ UPLOADCOLLECTIONITEM_RETRY_BUTTON_TEXT=Retry
 UPLOADCOLLECTIONITEM_EDIT_BUTTON_TEXT=Edit
 
 #XMSG: Message text for no data text in the UploadCollection
-UPLOADCOLLECTION_NO_DATA_TEXT=No files found.
+UPLOADCOLLECTION_NO_DATA_TEXT=No files found
+
+#XMSG: Message text for file upload error in the UploadCollection
+UPLOADCOLLECTION_UPLOAD_ERROR_TEXT=We''re having trouble uploading
+
+#XMSG: Message text for the description of file upload error in the UploadCollection
+UPLOADCOLLECTION_UPLOAD_ERROR_DESCRIPTION=Check your internet connection. If that''s not it, check the format and file size. If that still doesn''t help, contact your admins
```

Comment:
Maybe we should provide a distinct error message for the file-size/format error?
Otherwise I suggest:
Check your Internet connection. If that doesn’t help, check the file format and file size, or contact technical support.